liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -43,16 +46,22 @@
     private static File serverFile;
     
     private static Set<String> locations;
+    private static Properties props;
     
     public Set<String> getLocations() {
         return locations;
     }
     
+    public static Properties getProperties() {
+        return props;
+    }
+    
     private static File getServerFile() {
         return serverFile;
     }
     
     public ServerConfigDocument(File serverXML, File configDir) {
+        initializeProperties(serverXML, configDir);
```

Comment:
can you refactoring initializeProperties() and initializeAppsLocation() method?  They each go thru the same cycle of parsing server.xml and server include xml file.